.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,43 @@
-### Description
+## Description
+<!-- Please include a summary of the change, including which issue it fixes or what feature it adds. Include relevant motivation, context and documentation as appropriate. List dependencies that are required for this change, if any. -->
 
-### Test Plan
-<!-- Please provide us with clear details for verifying that your changes work. -->
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Performance improvement
+- [ ] Refactoring
+- [ ] Dependency update
+- [ ] Documentation update
+
+## Which Component or System Does This Change Impact?
```

Comment:
Can we add an option to 1) indicate if the change is a breaking change and 2) specify where/what does it?

Reply:
1. added `Breaking change` in the paragraph above.
2. would that go into the `Description`?

Reply:
1. LGTM
2. I just realized where/what the break change breaks is covered in this section. So, no need to add to `Description`.